src/main/controllers/RespondToTribunalResponseController.ts

```diff
@@ -75,16 +75,18 @@ export default class RespondToTribunalResponseController {
     if (error) {
       req.session.errors = [];
       req.session.errors.push(error);
-      return res.redirect(
-        `/${TranslationKeys.RESPOND_TO_TRIBUNAL_RESPONSE}/${req.params.appId}${getLanguageParam(req.url)}`
-      );
+      const pageUrl = `/${TranslationKeys.RESPOND_TO_TRIBUNAL_RESPONSE}/${req.params.appId}${getLanguageParam(
+        req.url
+      )}`;
```

Comment:
minor: declaring a languageParam const further up and reusing this would shorten this line so that it doesn't span 3 lines